lib/factory-helper/timestamps.rb

```diff
@@ -0,0 +1,31 @@
+module FactoryHelper
+  class Timestamps < Base
+
+    class << self
+      def between(from = ::Time.at(0), to = ::Time.at(2147483647))
+        random_time(from, to)
+      end
+
+      def forward(from = ::Time.at(0), seconds = 60)
+        to = from + seconds
+        random_time(from, to)
+      end
+
+      def backward(from = ::Time.at(0), seconds = 60)
+        to = from - seconds
+        random_time(to, from)
```

Comment:
here no, because `random_time` will convert the time difference to `nsec integer` (from floating point seconds) and uses rounding there.